src/cf-layout/src/cf-layout.less

```diff
@@ -346,16 +346,35 @@
 
     &__flush-top {
         margin-top: 0 !important;
+
+        &.block__border,
+        &.block__border-top {
+            border-top: none;
+        }
     }
 
     &__flush-bottom {
         margin-bottom: 0 !important;
+
+        &.block__border,
+        &.block__border-bottom {
+            border-bottom: none;
+        }
     }
 
     &__flush-sides {
         margin-right: -(@grid_gutter-width / 2);
         margin-left: -(@grid_gutter-width / 2);
 
+        .respond-to-max(@bp-xs-max, {
```

Comment:
Ah ok, thanks for clearing that up. Mind updating the one I linked to use `none` so it's consistent?